internal/client/client.go

```diff
@@ -105,10 +103,22 @@ type Client struct {
 	PartitionService     *icluster.PartitionService
 	ClusterService       *icluster.Service
 	name                 string
-	shutdownHandlers     []ShutdownHandler
+	ShutdownHandlers     []func(ctx context.Context)
```

Comment:
Why was `ShutdownHandlers` exported?

Reply:
It was the typo. It should not be exported. Fixed, [17f8ebc](https://github.com/hazelcast/hazelcast-go-client/pull/772/commits/17f8ebc0e27f5a5d71baf61a7eb5b9f5253c25bf).